app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -259,11 +254,11 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
     @Override
     public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        plusClient = new Plus.Builder(mTransport, mJsonFactory, null)
-                .setGoogleClientRequestInitializer(
-                        new CommonGoogleJsonClientRequestInitializer(BuildConfig.IP_SIMPLE_API_ACCESS_KEY))
-                .setApplicationName("GDG Frisbee")
-                .build();
+//        plusClient = new Plus.Builder(mTransport, mJsonFactory, null)
+//                .setGoogleClientRequestInitializer(
+//                        new CommonGoogleJsonClientRequestInitializer(BuildConfig.ANDROID_SIMPLE_API_ACCESS_KEY))
+//                .setApplicationName("GDG Frisbee")
+//                .build();
```

Comment:
I've tried lots of things in this work. 
I tried to get rid of these mTransport, mJsonFactory etc. But at the end I give up. 
These are left by mistake. Fixing now.